spartan/aztec-network/files/config/config-validator-env.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 set -eu
 
-# Pass the bootnode url as an argument
-# Ask the bootnode for l1 contract addresses
-output=$(node --no-warnings /usr/src/yarn-project/aztec/dest/bin/index.js get-node-info --node-url $1)
+# Pass a PXE url as an argument
+# Ask the PXE's node for l1 contract addresses
+output=$(node --no-warnings /usr/src/yarn-project/aztec/dest/bin/index.js get-node-info -u $1 --node-url '')
```

Comment:
```suggestion
output=$(node --no-warnings /usr/src/yarn-project/aztec/dest/bin/index.js get-node-info -u $1)
```

Reply:
No, that is the bug! See description.

Reply:
There's a default node url and a default PXE url and `get-node-info` prefer node url if it's defined (and since there's a default ...)

Reply:
ah right 😓